### ISSUE TITLE
Add caveats to management measure section based on Anderson et al. 2018 (if needed)

### ISSUE THREAD

**Issue by dtdougherty** [shcaba/FishPath-updates#218]
Review paper "How commercial fishing effort is managed" Anderson, et al. 2018

Add relevant cavets

**Comment by bsnouffer** [comment]
Natalie is doing a full review of management measures caveats. These comments will be taken into consideration during the review.

**Comment by NatalieDowling** [comment]
Thanks for this paper, Dawn. I have reviewed it as part of the full review of MMs I've just completed. This inspired various edits, including
- static caveats around "capital stuffing" for limited entry and other input controls.
- green recommendations for certain input controls for boom and bust species
- static caveats on catch limits re: inducing race to fish with TACs, but that these can be mitigated by individual fishing quotas, and noting that ITQs can have social impacts, but how these can be ameliorated.
- static caveat re: possible economic losses with spatial closures
Other points raised by the paper had, I felt, been embraced by existing caveats.
